source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+
+To get started with understanding metadata mapping files, first download a sample mapping file:
+
+.. TODO: Update this link
+.. download::
+   :url: https://data.qiime2.org/2017.5/tutorials/moving-pictures/sample_metadata.tsv
+   :saveas: sample-metadata.tsv
+
+This file can be opened and edited in a text editor, Google Sheets, or Keemei_. There is also a ``metadata tabulate`` visualization in QIIME 2 that allows for interactive sorting and filtering of metadata:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file sample-metadata.tsv
+     --o-visualization tabulated-sample-md.qzv
+
+.. question::
+   Based on the table in ``tabulated-sample-md.qzv``, how many samples are associated with Subject 1? How many samples are associated with the ``gut`` body site? Hint: use the search box and/or the column sorting options to assist with this query.
+
+Metadata from a QIIME 2 artifact
+--------------------------------
+
+As eluded to above, QIIME 2 also supports the notion of viewing artifacts as metadata (where it makes sense). An example of this is alpha-diversity output artifacts.
+
+To get started with understanding metadata mapping files, first download a sample mapping file:
+
+.. TODO: update this link
+.. download::
+   :url: https://docs.qiime2.org/2017.5/data/tutorials/moving-pictures/core-metrics-results/faith_pd_vector.qza
+   :saveas: faith_pd_vector.qza
+
+To view this artifact as metadata, simply pass it in to any method or visualizer that expects to see metadata (e.g. ``metadata tabulate``):
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file faith_pd_vector.qza
+     --o-visualization tabulated-faith-pd-md.qzv
+
+.. question::
+   What is the largest value of Faith's PD? What is the smallest? Hint: use the column sorting functions to assist with this query.
+
+Combining metadata
+------------------
+
+Because metadata can come from many different sources, QIIME 2 supports metadata merging when running commands. Building upon the examples above, simply passing ``--m-input-file`` multiple times will combine the metadata in the specified files:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file sample-metadata.tsv
+     --m-input-file faith_pd_vector.qza
+     --o-visualization tabulated-combined-md.qzv
+
+Merging is based on the identifiers specified in the first column of each metadata file (e.g. Sample IDs or Feature IDs). The resulting metadata after the merge represents the intersection of the identifiers in all of the specified files. Note, merging is non-destructive, and does not modify any of the input files.
+
+.. question::
+   Using another alpha diversity QZA, what happens when merging three files? How many columns are present in the resulting metadata file? How many of those columns represent the Sample IDs? How many of those columns represent alpha diversity metrics?
```

Comment:
I think you should not use `alpha diversity QZA`, but rather `SampleData[AlphaDiversity]` artifact, since they might not have a QZA (e.g., a power user working through this with the API).